### DISCUSSION TITLE
Monitor AdGuard Home with Zabbix

### DISCUSSION THREAD

**Discussion by diasdmhub** [AdguardTeam/AdGuardHome#6750] [Show and tell]
## Zabbix template for AdGuard Home​

If you're using Zabbix to monitor your network and want to collect statistics from your AdGuard Home instance, I've created a Zabbix template that provides some useful monitoring items.

The monitoring is agentless via a "REST-ish" API, since [AdGuard Home offers an OpenAPI specification](https://github.com/AdguardTeam/AdGuardHome/tree/master/openapi).
It is also available for a Zabbix agent for active monitoring behind blocking systems.
The main focus of the template is to monitor statistics from AdGuard Home, store them in the Zabbix historical database and create some observability.​

---

### [Check it out on GitHub](https://github.com/diasdmhub/AdGuard_Home_Zabbix_Template)

---

### Dashboad example:
![](https://github.com/diasdmhub/AdGuard_Home_Zabbix_Template/raw/main/images/dash_example1.png)

**Comment by Noschvie** [comment]
Wow!!! Great!
